src/main/java/net/javadiscord/javabot2/config/guild/ModerationConfig.java

```diff
@@ -16,6 +16,31 @@
 	 */
 	private long staffRoleId;
 
+	/**
+	 * The amount of seconds to be looked into the past to determine if a user is spamming.
+	 */
+	private int pastMessageCountBeforeDurationInSeconds;
```

Comment:
Maybe `spamMessageRetrospectionDurationSeconds` would more accurately describe what's happening. Right now this variable says nothing about the fact that it's for the spam filtering.